### ISSUE TITLE
[bug]: Won't download models like Deliberate

### ISSUE THREAD

**Issue by michael-murphree** [invoke-ai/InvokeAI#4121]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cpu

### VRAM

8GB

### What version did you experience this issue on?

3.0.1post3

### What happened?

During initial install, and from the WebUI attempting to have InvokerAI pull down the Deliberate model from HuggingFace results in an error. This happened both in the CUI during setup, and the WebUI just says "Unsupported Media Type"

Console output: 

```
[2023-08-01 15:37:25,194]::[uvicorn.access]::INFO --> 127.0.0.1:61404 - "POST /api/v1/models/import HTTP/1.1" 500
[2023-08-01 15:37:25,195]::[uvicorn.error]::ERROR --> Exception in ASGI application

Traceback (most recent call last):
  File "D:\apps\invoker\.venv\Lib\site-packages\uvicorn\protocols\http\httptools_impl.py", line 436, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\apps\invoker\.venv\Lib\site-packages\uvicorn\middleware\proxy_headers.py", line 78, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\apps\invoker\.venv\Lib\site-packages\fastapi\applications.py", line 270, in __call__
    await super().__call__(scope, receive, send)
  File "D:\apps\invoker\.venv\Lib\site-packages\starlette\applications.py", line 124, in __call__
    await self.middleware_stack(scope, receive, send)
  File "D:\apps\invoker\.venv\Lib\site-packages\starlette\middleware\errors.py", line 184, in __call__
    raise exc
  File "D:\apps\invoker\.venv\Lib\site-packages\starlette\middleware\errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "D:\apps\invoker\.venv\Lib\site-packages\starlette\middleware\cors.py", line 92, in __call__
    await self.simple_response(scope, receive, send, request_headers=headers)
  File "D:\apps\invoker\.venv\Lib\site-packages\starlette\middleware\cors.py", line 147, in simple_response
    await self.app(scope, receive, send)
  File "D:\apps\invoker\.venv\Lib\site-packages\fastapi_events\middleware.py", line 43, in __call__
    await self.app(scope, receive, send)
  File "D:\apps\invoker\.venv\Lib\site-packages\starlette\middleware\exceptions.py", line 79, in __call__
    raise exc
  File "D:\apps\invoker\.venv\Lib\site-packages\starlette\middleware\exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "D:\apps\invoker\.venv\Lib\site-packages\fastapi\middleware\asyncexitstack.py", line 21, in __call__
    raise e
  File "D:\apps\invoker\.venv\Lib\site-packages\fastapi\middleware\asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "D:\apps\invoker\.venv\Lib\site-packages\starlette\routing.py", line 706, in __call__
    await route.handle(scope, receive, send)
  File "D:\apps\invoker\.venv\Lib\site-packages\starlette\routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "D:\apps\invoker\.venv\Lib\site-packages\starlette\routing.py", line 66, in app
    response = await func(request)
               ^^^^^^^^^^^^^^^^^^^
  File "D:\apps\invoker\.venv\Lib\site-packages\fastapi\routing.py", line 235, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\apps\invoker\.venv\Lib\site-packages\fastapi\routing.py", line 161, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\apps\invoker\.venv\Lib\site-packages\invokeai\app\api\routers\models.py", line 155, in import_model
    installed_models = ApiDependencies.invoker.services.model_manager.heuristic_import(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\apps\invoker\.venv\Lib\site-packages\invokeai\app\services\model_manager_service.py", line 565, in heuristic_import
    return self.mgr.heuristic_import(items_to_import, prediction_type_helper)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\apps\invoker\.venv\Lib\site-packages\invokeai\backend\model_management\model_manager.py", line 1045, in heuristic_import
    installed = installer.heuristic_import(thing)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\apps\invoker\.venv\Lib\site-packages\invokeai\backend\install\model_install_backend.py", line 257, in heuristic_import
    models_installed.update({str(model_path_id_or_url): self._install_url(model_path_id_or_url)})
                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\apps\invoker\.venv\Lib\site-packages\invokeai\backend\install\model_install_backend.py", line 288, in _install_url
    dest = self.config.models_path / info.base_type.value / info.model_type.value / location.name
                                     ^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'base_type'
[2023-08-01 15:42:05,169]::[uvicorn.access]::INFO --> 127.0.0.1:61470 - "GET /api/v1/images/?is_intermediate=true HTTP/1.1" 200
unet\diffusion_pytorch_model.safetensors not found
Fetching 14 files:  36%|████████████████████████████████████████████████████████████████████████████▍                                                                                                                                         | 5/14 [00:01<00:01,  4.76it/s]
Downloading pytorch_model.bin:   2%|████▏                                                                                                                                                                                                | 10.5M/492M [00:00<00:19, 24.6MB/s]
Downloading (…)on_pytorch_model.bin:   0%|▌                                                                                                                                                                                             | 10.5M/3.44G [00:00<03:25, 16.7MB/s]
Downloading (…)on_pytorch_model.bin:   3%|█████▉                                                                                                                                                                                         | 10.5M/335M [00:00<00:17, 18.7MB/s]
[2023-08-01 15:44:00,500]::[InvokeAI]::ERROR --> Diffusers model XpucT/Deliberate could not be downloaded. Skipping.
[2023-08-01 15:44:00,500]::[InvokeAI]::WARNING --> Could not determine type of repo XpucT/Deliberate. Skipping install.
[2023-08-01 15:44:00,504]::[InvokeAI]::ERROR --> Import failed
```

**Comment by Millu** [comment]
This was fixed as 3.0.2

**Comment by chris-sadighgroup** [comment]
up, same issue for version 3.1.1
